RegenAllByteCodeNoBuild.cmd

```diff
@@ -0,0 +1,62 @@
+::-------------------------------------------------------------------------------------------------------
+:: Copyright (C) Microsoft. All rights reserved.
+:: Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+::-------------------------------------------------------------------------------------------------------
+
+:: WARNING: be careful when using this script as it assumes that
+:: you already have bytecode-format-compatible builds for all required flavors.
+:: This script helps speed things up when you are only making changes to

```

Comment:
nit: feels like something is missing at the end of this sentence.
`... making changes to scripts, e.g. Intl.js` ?